tests/main/ubuntu-core-create-user/task.yaml

```diff
@@ -7,6 +7,15 @@ restore: |
     sed -i '/^mvo/d' /var/lib/extrausers/group
     rm -rf /home/mvo
 execute: |
+    if [ "$MANAGED_DEVICE" = "true" ]; then
```

Comment:
Do we need this flag? Can't we create a user and remove the user on the test itself, thus making it a general test?